### PR TITLE
Support of Puppet FOSS for Webhook

### PR DESCRIPTION
**Pull request by acidprime**
This is a fix for #149

### CODE REVIEW

**Comment by mvisonneau** [comment]
Quicker than me :) Still working fine for me on Ubuntu 14.

File: templates/webhook.bin.erb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/opt/puppet/bin/ruby
+#!<%= if @is_pe == true or @is_pe == 'true' then '/opt/puppet/bin' else '/usr/bin' end %>/ruby
```
Comment (igalic) [inline templates/webhook.bin.erb]: this will fail when strict variables is enabled, cuz it doesn't actually exist…

Comment (mvisonneau) [inline templates/webhook.bin.erb]: Can we add the following in the webhook.pp to solve this problem ?

```puppet
if ! defined( $is_pe ) {
  $is_pe = false
}
```

Comment (igalic) [inline templates/webhook.bin.erb]: if i have [this discussion](https://github.com/puppetlabs/puppetlabs-concat/pull/270) another time, i'll scream :P

**Pull request merged**